app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -105,6 +106,20 @@ def self.params_for_create(ems)
     }
   end
 
+  def params_for_clone
+    {
+      :fields => [
+        {
+          :component  => 'text-field',
+          :name       => 'name',
+          :id         => 'name',
+          :label      => _('Volume Name'),
```

Comment:
I would prefer to call this `_('Volume Base Name')` and then add some "helperText" so the user can determine what the cloned volume name will be.
```ruby
:helperText => _("Base name of the new cloned volume(s). Cloned Volume names will be prefixed with 'clone-' and suffixed with '-#####' (where ##### is a 5 digit random number)")
```